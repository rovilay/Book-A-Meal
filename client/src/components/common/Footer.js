import React from 'react';

const Footer = () => (
  <footer className="footer">
    <div>
      <p>Book-A-Meal, copyright &copy; 2018</p>
    </div>
  </footer>
);

export default Footer;
