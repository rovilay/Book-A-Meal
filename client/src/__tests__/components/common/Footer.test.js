import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../../components/common/Footer';


describe('Footer component test', () => {
  it('should render Footer correctly', (done) => {
    const wrapper = shallow(<Footer />);

    expect(wrapper).toMatchSnapshot();

    done();
  });
});

