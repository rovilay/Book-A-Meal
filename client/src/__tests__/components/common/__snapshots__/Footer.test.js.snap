// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component test should render Footer correctly 1`] = `
<footer
  className="footer"
>
  <div>
    <p>
      Book-A-Meal, copyright Â© 2018
    </p>
  </div>
</footer>
`;
